app/components/Entrace.vue

```diff
@@ -0,0 +1,29 @@
+<!-- TODO: 
```

Comment:
Let's not have TODO's and ideas in the code but lets create project/issues for those. Then just add a link to that issue to this file

Reply:
Issues created.